src/events/index.js

```diff
@@ -38,6 +38,31 @@ import { formatPaginatedDataFromServer } from '../utils/pagination';
  * @property {string} updatedAt ISO 8601 Extended Format date/time string
  */
 
+/**
+ * @typedef {Object} EventUser
+ * @property {string} createdAt ISO 8601 Extended Format date/time string
+ * @property {string} email
+ * @property {string} firstName
+ * @property {string} id
+ * @property {boolean} isMachineUser
+ * @property {Array} [IOSDevices]
```

Comment:
I do not know if this is the correct syntax to show that the api return an array of objects back.

Reply:
I've had time to think about it. I think this should be {Object} but I do not know how to account for the array part